docs/conf.py

```diff
@@ -313,3 +319,31 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
+
+link_files = {
+    f: dict(
+        using=dict(
+            GH='https://github.com',
+        ),
+        replace=[
+            dict(
+                pattern=r'((Issue|PR)\s?)?#(?P<issue_or_pr>\d+)',
+                url='{GH}/aio-libs/{project}/issues/{issue_or_pr}',
```

Comment:
I would suggest to use extlinks instead like [this](https://github.com/apache/couchdb-documentation/blob/master/src/conf.py#L114). It's better for point of markup.

Reply:
Fair enough, but would it render nicely at github.com or pypi.org?

Reply:
No, it wouldn't. But I wouldn't care about this as well.

Reply:
@kxepal https://github.com/aio-libs/multidict/blob/master/setup.py#L77 `CHANGES.rst` is included there. We can't just break PYPI page in favor of SphinX, unless @asvetlov approves dropping it.